index.js

```diff
@@ -33,39 +35,55 @@ const onHeadingClick = (x, activeTab, i, setState, onClick) => () => {
 	}
 };
 
-const getHeadings = ({items, onClick}, {activeTab}, setState) => {
-	return items.map((x, i) => (
-		<div class={['Tabs-heading', {'Tabs-heading--active': activeTab === i}]} onClick={onHeadingClick(x, activeTab, i, setState, onClick)}>
-			{x.heading}
-		</div>
-	));
-};
+const getHeadings = ({items, onClick}, {activeTab}, setState) => items.map((x, i) => (
+	<div
+		key={i} // eslint-disable-line react/no-array-index-key
+		class={['Tabs-heading', {'Tabs-heading--active': activeTab === i}]}
+		onClick={onHeadingClick(x, activeTab, i, setState, onClick)}
+	>
+		{x.heading}
+	</div>
+));
 
-const getTabs = ({items}, {activeTab}) => {
-	return items.map(({content}, i) => (
-		<div class={['Tab', {'Tab--active': activeTab === i}]} style={{display: activeTab === i ? 'block' : 'none'}}>
-			{content}
-		</div>
-	));
+const getTabs = ({items}, {activeTab}) => items.map(({content}, i) => (
+	<div
+		key={i} // eslint-disable-line react/no-array-index-key
+		class={['Tab', {'Tab--active': activeTab === i}]}
+		style={{display: activeTab === i ? 'block' : 'none'}}
+	>
+		{content}
+	</div>
+));
+
+const setActive = (items, setState) => {
+	for (const [i, x] of items.entries()) {
+		if (x.active) {
+			setState({activeTab: i});
+		}
+	}
 };
 
-const setActive = (items, setState) => items.forEach((el, i) => el.active === true && setState({activeTab: i}));
+const afterMount = ({props}, el, setState) => {
+	setActive(props.items, setState);
+};
```

Comment:
Since this and `afterUpdate` doesn't `return` it's confusing if they are arrow functions.